versions/3.0.md

```diff
@@ -1102,10 +1102,10 @@ application/json:
   schema:
```

Comment:
Capturing a TDC discussion comment "I would expect that any tooling worth its salt would throw this out if it failed to parse, but that it would try to parse it."